src/integrations/blocks/abstract-dynamic-block-v3.php

```diff
@@ -75,4 +75,21 @@ public function register_block() {
 	 * @return string The block output.
 	 */
 	abstract public function present( $attributes );
+
+	/**
+	 * This is needed because when the editor is loaded in an Iframe the link needs to open in a different browser window.
```

Comment:
```suggestion
	/**
	 * Checks whether the links in the block should have target="blank".
	 *
	 * This is needed because when the editor is loaded in an Iframe the link needs to open in a different browser window.
```